virt/virtual_machines/virt-configuring-pci-passthrough.adoc

```diff
@@ -0,0 +1,24 @@
+[id="virt-configuring-pci-passthrough"]
+= Configuring PCI passthrough
+include::modules/virt-document-attributes.adoc[]
+:context: virt-configuring-pci-passthrough
+
+toc::[]
+
+//This assembly contains the content for
+//configuring PCI passthrough by using the CLI. There are
+//plans to enable PCI passthrough configuration
+//by using the web console (next release).
+//When this feature is available in the web console, please
+//add the new content to this assembly.
+
+The PCI Passthrough feature enables you to access and manage hardware devices from a virtual machine. When PCI Passthrough is configured, the PCI devices function as if they were physically attached to the virtual machine.
+
+To configure a hardware device as a host device for PCI passthrough, complete these tasks:
```

Comment:
I'm not seeing where to find this and the module includes in the preview build?

Reply:
The virt-about-pci-passthrough module was not listed in the assembly, therefore this content did not show up. Fixed the assembly file and listed the module.